docs/src/tutorials/exporting.md

```diff
@@ -1,51 +1,173 @@
 ---
 title: "Exporting data"
-description: |
-  Platform.sh aims to be a great host, but we never want to lock you in to our service. Your code and your data belong to you, and you should always be able to download your site's data for local development, backup, or to "take your data elsewhere".
+description: See how to export your code, files and service data.
 ---
 
-{{% description %}}
+As a Platform.sh user, your code and data belong to you.
+At any time, you can download your site's data for local development, to back up your data, or to change provider.
+## Before you begin
 
-## Downloading code
+You need:
 
-Your application's code is maintained in Git.
-Because Git is a distributed system,
-you can download your entire code history by running `git clone` or `platform get`.
+- [Git](https://git-scm.com/downloads)
+- A Platform.sh account
+- Code in your project
+- Optional: the [Platform.sh CLI](../administration/cli/_index.md)
 
-## Downloading files
+## 1. Download your app's code
 
-To download files from your built app,
-you can [set up mounts](../create-apps/app-reference.md#mounts) 
-and run commands using the Platform.sh CLI.
-You can also use an SSH client such as `rsync` or `scp`.
+Your app's code is maintained through the Git version control system.
 
-For more information, see how to [transfer files to and from a built app](../development/file-transfer.md).
+To download your entire app's code history:
 
-## Download data from services
+{{< codetabs >}}
+
+---
+title=Using the CLI
+file=none
+highlight=false
+---
+
+1. List all your projects with:
+
+   ```bash
+   platform projects
+   ```
+
+2. Retrieve the project you want to back up with:
+
+   ```bash
+   platform get {{< variable "PROJECT_ID" >}}
+   ```
+
+<--->
+
+---
+title=Using Git
+file=none
+highlight=false
+---
+
+1. In the [Console](https://console.platform.sh/), open your project and click **Code {{< icon chevron >}}**.
+2. Click **Git**.
+3. To copy the command, click **{{< icon copy >}}**.
+4. To clone the repository, run a command similar to the following:
+
+   ```text
+   git clone abcdefgh1234567@git.eu.platform.sh:abcdefgh1234567.git project-name
+   ```
+
+{{< /codetabs >}}
+
+## 2. Download your files
+
+Some files might not be stored in Git,
+such as data your app writes [in mounts](../create-apps/app-reference.md#mounts).
+
+To download your files:
+
+{{< codetabs >}}
```

Comment:
:question: Could the entire step could be replaced by links to relevant sections of the transfer files page? Both ways are explained there.